ops/Makefile

```diff
@@ -17,6 +18,22 @@ default:
 .valid-env-%:
 	@case $* in test|dev|demo|training|pentest|stg|prod) ;; *) echo "$* is not a valid environment"; exit 1;; esac
 
+api.tfvars: /dev/null
+	echo "acr_image_tag=\"$(DEPLOYED_COMMIT)\"" > $@; \
+	echo "deploy_workflow=\"${GITHUB_WORKFLOW}\"" >> $@; \
+	if [[ "release" == "$(GITHUB_EVENT_NAME)" ]]; \
+		then echo "deploy_tag=\"$(subst refs/tags/,,$(GITHUB_REF))\"" >> $@;\
+	fi; \
+	echo "deploy_runnumber=${GITHUB_RUN_NUMBER}" >> $@; \
+	echo "deploy_timestamp=\"$(shell date +%Y-%m-%dT%H:%M:%S%z) \"" >> $@; \
+	echo "deploy_actor=\"$(GITHUB_ACTOR)\"" >> $@;
+
+init-%: .valid-env-%
+	cd $*; terraform init
+
+deploy-%-api: .valid-env-% api.tfvars
+	cd $*; terraform apply -auto-approve -var-file=../api.tfvars -target="module.simple_report_api"
```

Comment:
```suggestion
	terraform -chdir=$*apply -auto-approve -var-file=../api.tfvars -target="module.simple_report_api"
```

Reply:
I think `--chdir` is a v0.15+ flag - are we sure we're using that in the pipeline at this point? It's been out for a while so I'm assuming yes.

Reply:
We are not—all of our terraform-setup actions are currently  being passed `terraform_version: 0.14.10`. I tested this change this morning and it did work in github as well as locally, so I think we're in the clear.

Reply:
See also #1371 🙂